src/lib.rs

```diff
@@ -35,6 +39,10 @@ impl Plugin for VelloGraphicsPlugin {
                 build_vector::<VelloCircle>(),
                 build_vector::<VelloLine>(),
                 build_vector::<VelloBezPath>(),
+                build_head::<VelloRect>(),
+                build_head::<VelloCircle>(),
+                build_head::<VelloLine>(),
+                build_head::<VelloBezPath>(),
```

Comment:
We can have `build_head` systems added into `build_vector` so we wouldn't need to rewrite twice.

Reply:
will be resolved in the next commit!